### ISSUE TITLE
[LANGUAGE] Colors in lists in non-English don't work

### ISSUE THREAD

**Issue by Felienne** [hedyorg/hedy#3356]
As mentioned in the recent turtle PR, colors in non-English don't work as well as they do in English.

> In addition to that I also found that colors, especially those in lists, don't work when you're not using English color names. I don't know how easy/hard it is to solve that, but it should be a different ticket/PR.

_Originally posted by @arjank in https://github.com/Felienne/hedy/issues/3332#issuecomment-1259957753_

This works in level 3, Dutch keywords but English color names:

![image](https://user-images.githubusercontent.com/1003685/193224178-f6bc62d6-aad7-45ad-9988-ac575bd2a2b7.png)

```
lijstkleuren is white, pink, green, blue
kleur lijstkleuren at random
vooruit 150
```

But, with Dutch color names, the same does not work:
![image](https://user-images.githubusercontent.com/1003685/193223938-0d0d9995-f3d4-4129-b5d3-8cfafc649438.png)

```
lijstkleuren is wit, roze, groen, blauw
kleur lijstkleuren at random
vooruit 150
```

**Comment by Felienne** [comment]
Hi @arjank! I think this was the issue you meant on #3332 right?

**Comment by arjank** [comment]
Yes, this is what I meant.

**Comment by Felienne** [comment]
Oops, the fix in #4675 caused #4752, so I am reverting that for now, will pick up the issue later in #4751 where I started the partial work
